### PR TITLE
Topologymap

### PR DESCRIPTION
**Pull request by henktillman**
Creates geographical visualization of RippleD node locations. Nodes are placed on a map if their IP address (and thus coordinates) is available.

### CODE REVIEW

File: Gruntfile.js [hunk 1]
```diff
@@ -9,7 +9,7 @@ module.exports = function (grunt) {
   grunt.loadNpmTasks('grunt-contrib-jshint');
   grunt.loadNpmTasks('grunt-contrib-concat');
   grunt.loadNpmTasks('grunt-contrib-watch');
-  grunt.loadNpmTasks('grunt-contrib-uglify');
```
Comment (shekenahglory) [inline Gruntfile.js]: this is a problem - uglify isn't used in dev mode but to compile for production its necessary - why did you need to take it out?

Comment (henktillman) [inline Gruntfile.js]: My mistake - it makes development much faster. When you run grunt to compile on your local machine, with uglify it's about 20 seconds, without about 3 seconds. I'll put it back in.

Comment (shekenahglory) [inline Gruntfile.js]: run `grunt watch` and it will not compile and it will reload when you make
changes, just navigate to the '/build' directory instead of '/bin'

**Comment by shekenahglory** [comment]
LGTM
